.teamcity/settings.kts

```diff
@@ -79,6 +79,7 @@ object Build_Release: BuildType({
         param("env.RG_DEV", "ESdCDPSBDMK8SDev")
         param("env.TARGET", "main")
         param("env.BRANCH", "main")
+        param("env.NEXT_CONTENT_GRAPHQL", "%vault:dts-secrets-dev/scrumpoker!/NEXT_CONTENT_GRAPHQL%")
```

Comment:
I don't know if it is case sensitive but if it is, it should be `scrumPoker` instead of `scrumpoker`.